internal/store/postgres/job/job_repository.go

```diff
@@ -489,8 +489,8 @@ func (j JobRepository) ReplaceUpstreams(ctx context.Context, jobsWithUpstreams [
 	}
 
 	var jobFullName []string
-	for _, upstream := range storageJobUpstreams {
-		jobFullName = append(jobFullName, upstream.getJobFullName())
+	for _, jobWithUpstream := range jobsWithUpstreams {
+		jobFullName = append(jobFullName, jobWithUpstream.Job().FullName())
 	}
 
 	if err = j.deleteUpstreams(ctx, tx, jobFullName); err != nil {
```

Comment:
can we rename this function to `.deleteUpstreamsByJobName` ? to avoid confusion which the scope of this function is to delete the upstream from given `job_name`

Reply:
updated to `deleteUpstreamsByJobNames`